### PR TITLE
Create tools pod on FaaS consumer

### PR DESCRIPTION
**Pull request by jilju**
Create tools pod on consumer cluster in FaaS configuration.

### CODE REVIEW

**Comment by jilju** [comment]
Verified: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-odf-multicluster/1703/
Tools pod created on consumer cluster.

**Review by ebondare — APPROVED** [review]

File: ocs_ci/helpers/managed_services.py [hunk 1]
```diff
@@ -694,3 +694,71 @@ def verify_faas_provider_storagecluster_storages(sc_data):
 
     log.info(f"The values in the storage profile {default_sp_name} are correct")
     log.info("Finish Verifying all the storages in the provider faas storagecluster")
+
+
+def create_toolbox_on_faas_consumer():
```
Comment (yitzhak12) [inline ocs_ci/helpers/managed_services.py:699]: In functions when we switch from one cluster to another, we had better use the decorator `@switch_to_orig_index_at_last` from the file https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/utility/decorators.py.

Comment (jilju) [reply ocs_ci/helpers/managed_services.py:701]: Added the decorator.

**Review by yitzhak12 — APPROVED** [review]

File: tests/conftest.py [hunk 2]
```diff
@@ -5650,6 +5651,30 @@ def patch_consumer_toolbox_with_secret():
     config.switch_ctx(restore_ctx_index)
 
 
+@pytest.fixture(scope="session", autouse=True)
+def toolbox_on_faas_consumer():
+    """
+    Create tools pod on FaaS consumer cluster at the start of a test run while using multicluster configuration
+
+    """
+    from ocs_ci.helpers.managed_services import create_toolbox_on_faas_consumer
```
Comment (fbalak) [inline tests/conftest.py:5660]: The import should be at the beginning of the file.

**Review by fbalak — APPROVED** [review]

File: tests/conftest.py (hunk 2 above)
Comment (jilju) [reply tests/conftest.py:5660]: Added here to avoid circular import

**Pull request merged**